.github/workflows/update_disabled_tests.yml

```diff
@@ -18,7 +18,12 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Generate new disabled tests and jobs jsons
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          # Octokit is used to confirm issue creator's admin permission before
+          # allowing them to disable jobs
+          python3 -mpip install octokitpy==0.15.0
```

Comment:
if you want to remove this dependency, request or open_url like in the python scripts in pytorch also work

Reply:
Noted.  It does make the code looks easier to read IMO (at the cost of having one more dependency).  Let me try to draft a version without it, and compare the twos.